arches/app/media/css/pages/_landingpage.scss

```diff
@@ -1,75 +1,5 @@
 .navbar-toggle {
```

Comment:
I don't see this class used anywhere.  Can we just get rid of it?

Reply:
@apeters for screen sizes < desktop, the navbar is pushed into a dropdown:
<img width="1341" alt="Screenshot 2023-10-16 at 7 16 45 PM" src="https://github.com/archesproject/arches/assets/22161487/53e4d297-0da8-45e4-bb58-710edb627b8c">